pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I haven't used Gradle much, so it was easier for me to convert this to a Maven project. You don't have to use it, but this is what I did to get everything running.